pyintelowl/pyintelowl.py

```diff
@@ -107,6 +107,7 @@ def send_file_analysis_request(
         force_privacy: bool = False,
         private_job: bool = False,
         disable_external_analyzers: bool = False,
+        tags=List[int],
```

Comment:
this is not correct. First, you should put this import to the very last and it should be like `tags: List[int] = []`